src/bundle/Resources/public/js/scripts/admin.contenttype.relation.default.location.js

```diff
@@ -35,6 +41,21 @@
             restInfo: { token, siteaccess }
         }, config)), udwContainer);
     };
+    const toggleResetStartingLocationBtn = (button, isEnabled) => {
+        isEnabled ? button.removeAttribute('disabled') : button.setAttribute('disabled', true);
```

Comment:
We are using thernary more for assignment, not for invoking functions.
I would see it should something like this:
```js
    const methodName = isEnabled ? 'removeAttribute' : 'setAttribute';
    
    button[methodName]('disabled', 'disabled');
```

Reply:
Actually, `toggleAttribute` was already used by Darek in admin-ui:
https://github.com/ezsystems/ezplatform-admin-ui/pull/1847/files#diff-9530ed0ec9743cd8231648df7312b0eaf421d4b5375c3217fab6249b5a3b1c54R109

Reply:
Not sure about `toggleAttribute` but please note, that we are dealing with `1.5` AdminUi code, so I think it would be good to stay aligned and eventually improve this part starting from `2.3` onwards.

Reply:
`toggleAttribute` I wouldn't use as this PR targets 2.5.

So either the way I suggested or the `regular if`. Both are fine for me. If @tischsoic hates this so much you can use the regular if.